xml/depl_smt_setup.xml

```diff
@@ -258,8 +253,7 @@ done</screen>
    <title>Mirroring the Repositories</title>
    <para>
     Repositories added to &smt; are updated automatically via cron
-    job. However, to make repositories that have been added available, a
-    manual mirroring run needs to be triggered initially. Do so by running the
+    job. You should update them immediately after adding them by running the
```

Comment:
I would even put it stronger: To make them available, update them immediately...